zebedee-cms/src/main/java/com/github/onsdigital/zebedee/Zebedee.java

```diff
@@ -85,6 +81,7 @@ public class Zebedee {
     private final DatasetService datasetService;
     private final ImageService imageService;
     private final KafkaService kafkaService;
+    private final StaticFilesService staticFilesService;
```

Comment:
Please return the single import 
(if you are using intellij push and commit in terminal outside ide)